File: messages/src/main/java/com/thundashop/core/accountingmanager/AccountingTransferConfig.java
package com.thundashop.core.accountingmanager;

import com.thundashop.core.common.DataCommon;
import java.util.HashMap;
import java.util.List;

public class AccountingTransferConfig extends DataCommon {
    public Integer delay;
    public Integer includeUsers;
    public List<AccountingTransferConfigTypes> paymentTypes;
    public TransferFtpConfig ftp;
    public String transferType;
    public String orderFilterPeriode = "";
    public String subType = "orders";
    public HashMap<String, String> paymentTypeCustomerIds = new HashMap();
    public String username;
    public String password;
    public Integer startCustomerCodeOffset;
}

File: html5apps/terminal/events/core/accountingmanager/AccountingTransferConfig.php
<?php
class core_accountingmanager_AccountingTransferConfig extends core_common_DataCommon  {
	/** @var String */
	public $delay;

	/** @var String */
	public $includeUsers;

	/** @var core_accountingmanager_AccountingTransferConfigTypes[] */
	public $paymentTypes;

	/** @var core_accountingmanager_TransferFtpConfig */
	public $ftp;

	/** @var String */
	public $transferType;

	/** @var String */
	public $orderFilterPeriode;

	/** @var String */
	public $subType;

	/** @var String */
	public $paymentTypeCustomerIds;

	/** @var String */
	public $username;

	/** @var String */
	public $password;

	/** @var String */
	public $postingDateType;

	/** @var String */
	public $startCustomerCodeOffset;

}
?>